src/bin/vip-logs.js

```diff
@@ -232,31 +232,39 @@ command( {
 	envContext: true,
 	module: 'logs',
 } )
-	.option( 'type', 'The type of logs to be returned: "app" or "batch"', 'app' )
+	.option(
+		'type',
+		'Specify the type of Runtime Logs to retrieve. Accepts "batch" (only valid for WordPress environments) or "app" (default).'
+	)
 	// The default limit is set manually in the validateInputs function to address validation issues, avoiding incorrect replacement of the default value.
-	.option( 'limit', `The maximum number of log lines (defaults to ${ LIMIT_DEFAULT })` )
-	.option( 'follow', 'Keep fetching new logs as they are generated' )
-	.option( 'format', 'Output the log lines in CSV or JSON format', 'table' )
+	.option(
+		'limit',
+		`The maximum number of entries to return. Accepts an integer value between 1 and 5000 (defaults to ${ LIMIT_DEFAULT }).`
+	)
+	.option( 'follow', 'Output new entries as they are generated.' )
+	.option(
+		'format',
+		'Render output in a particular format. Accepts "table" (default), "csv", and "json".'
```

Comment:
Related to what @sjinks [pointed out above](https://github.com/Automattic/vip-cli/pull/2089/files#r1852626576), we lost the 3rd argument here

```suggestion
		'Render output in a particular format. Accepts "table" (default), "csv", and "json".', 'table'
```

Reply:
Addressed here: https://github.com/Automattic/vip-cli/pull/2089/files#diff-2842aeea5897751928ed910d842cd542768c354d43d746f1877afab1869eac9fR26-R28